modules/virt-configuring-storage-class-bootsource-update.adoc

```diff
@@ -39,9 +39,13 @@ spec:
       template:
         spec:
           storageClassName: <new_storage_class> <1>
-#...
+      schedule: "0 */12 * * *" <2>
```

Comment:
this section is about changing the storage class for boot images. how come we suddenly want to include the schedule/managedDataSource field?

Reply:
@akalenyu  If I am reading this right, in order to add `spec.dataImportCronTemplates.spec.template.spec.storageClassName` parameter, we need to add the `schedule` and `managedDataSource` parameters. Am I doing something wrong? 

```
# Please edit the object below. Lines beginning with a '#' will be ignored,
# and an empty file will abort the edit. If an error occurs while saving this file will be
# reopened with the relevant failures.
#
# hyperconvergeds.hco.kubevirt.io "kubevirt-hyperconverged" was not valid:
# * spec.dataImportCronTemplates[0].spec.managedDataSource: Required value
# * spec.dataImportCronTemplates[0].spec.schedule: Required value
```

Reply:
ah you're right, this was an actual doc bug then